src/app/projects/components/project-settings/project-settings.component.ts

```diff
@@ -952,4 +955,8 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy, AfterViewIni
   setLabelingTaskTarget(id: string) {
     this.settingModals.labelingTask.create.taskId = id;
   }
+
+  updateVisibilityAttributes(value: string, attributeContainer: any) {
+    this.projectApolloService.updateAttribute(this.project.id, attributeContainer.get("id").value, attributeContainer.get("dataType").value, attributeContainer.get("isPrimaryKey").value, attributeContainer.get("name").value, attributeContainer.get("sourceCode").value, value).pipe(first()).subscribe();
+  }
```

Comment:
might be mistaken but dont we only need to change the value of the group element to trigger the valueChanges subscription?